batch/batch/driver/scheduler.py

```diff
@@ -133,7 +137,8 @@ def allocate_cores(user, mark):
             '''
 SELECT user
 FROM user_resources
-WHERE running_cores_mcpu > 0;
+GROUP BY user
+HAVING CAST(SUM(running_cores_mcpu) AS SIGNED) > 0;
```

Comment:
What's the effect of the `CAST(... AS SIGNED)`? It looks like running_cores_mcpu is already signed, right? Does the `sum` change that somehow?

Reply:
The sum makes it a float/decimal rather than an int. I figured it was good to cast it to an int in case there's floating point imprecision. Not sure it makes a difference honestly. I can take it out.